src/main/java/ro/developmentfactory/thedrones/controller/dto/DroneStatusResponse.java

```diff
@@ -11,12 +12,23 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @Builder
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class DroneStatusResponse {
     private UUID idDroneStatus;
     private Integer currentPositionX;
     private Integer currentPositionY;
     private Direction facingDirection;
     private UUID idDrone;
+    private TurnDirection turnDirection;
+
+    public DroneStatusResponse(UUID idDrone) {
+        this.idDrone = idDrone;
+    }
+    public enum TurnDirection {
```

Comment:
I'd recommend moving this `TurnDirection` to a separate enum (not a response!), you can put it to dtos in controller package. then you can use it as `@PathVariable`, see more [here](https://www.baeldung.com/spring-enum-request-param)